### ISSUE TITLE
undefined errors caused by getPeerStatistics and setPeerListener

### ISSUE THREAD

**Issue by provstevi** [priologic/easyrtc#253]
The calls to the Firefox and Chrome methods in easyrtc.getPeerStatistics() are being made incorrectly, causing undefined errors when trying to use it. 

Also had an issue with the sendAnswer function definition being too deeply nested to access the peerConns array, causing another undefined error; moving that up resolved the problem. 

Will submit a PR that addresses both these issues.

**Comment by EricDavies** [comment]
Are you using beta branch or master?

Can you describe under what circumstances you saw a problem with the sendAnswer function? As far as I can see, it seems to work with the demo code and isn't producing any error messages. 

I haven't looked at the getPeerStatistics methods recently will check them out.

**Comment by EricDavies** [comment]
With regards to the getPeerStatistics methods, getPeerStatistics seems to work properly under both firefox and chrome. I modified the performCall method in the simple audio video demo to be as follows:

function performCall(otherEasyrtcid) {
    easyrtc.hangupAll();
    var successCB = function() {
         function statscallback (peerid, data) {
            console.log("stats for peer " + peerid);
            console.log(data);
         }
         setTimeout( function() {
         easyrtc.getPeerStatistics(otherEasyrtcid, statscallback );
            }, 5000);
    };

    var failureCB = function() {};
    easyrtc.call(otherEasyrtcid, successCB, failureCB);
}

What is broken is a variable called standardStatsFilter that should be getting exported as a member of easyrtc and isn't. But it just defines an optional third argument value for the getPeerStatistics method (albeit one that is quite convenient to have).

**Comment by provstevi** [comment]
Yes, I'm seeing you'd already made the fix I submitted on the beta branch, so I'll close this. 

I am still occasionally seeing the undefined error thrown in the sendAnswer call, but I cannot reliably reproduce it, and moving the code at this point isn't as trivial as it was with the 1.7 version, so I'll just keep an eye out for it. We haven't seen it in our working project, I only saw it as I was testing out using the demo code. 

Any idea when you'll be releasing a build?
